h2o-core/src/main/java/water/JettyHTTPD.java

```diff
@@ -305,7 +305,13 @@ public void registerHandlers(HandlerWrapper handlerWrapper) {
     ServletContextHandler context = new ServletContextHandler(
         ServletContextHandler.SECURITY | ServletContextHandler.SESSIONS
     );
-    context.setContextPath("/");
+
+    if(null != H2O.ARGS.context_path && !H2O.ARGS.context_path.isEmpty()) {
+      context.setContextPath(H2O.ARGS.context_path);
```

Comment:
`/H2O` is fine, i just wanted to make sure that the web server is correctly configured even user passes `H2O`.